src/main/java/com/homeworks/factory_methodd/SingeltonCreator.java

```diff
@@ -0,0 +1,11 @@
+package com.homeworks.factory_methodd;
```

Comment:
https://mate-academy.github.io/style-guides/java/java.html#s5.2.1-package-names
Package names are all lowercase, with consecutive words simply concatenated together (no underscores). For example, com.example.deepspace, not com.example.deepSpace or com.example.deep_space.